src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewKernel.php

```diff
@@ -40,4 +46,34 @@ public function getRootDir()
 
         return $this->rootDir;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLogDir()
+    {
+        $context = $this->getContext();
+
+        // emulating own context for directory creation, but still use the original one afterwards
+        $this->setContext('preview');
+        $logDirectory = parent::getLogDir();
+        $this->setContext($context);
+
+        return $logDirectory;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCacheDir()
+    {
+        $context = $this->getContext();
+
+        // emulating own context for directory creation, but still use the original one afterwards
+        $this->setContext('preview');
+        $cacheDirectory = parent::getCacheDir();
+        $this->setContext($context);
+
+        return $cacheDirectory;
+    }
```

Comment:
@chirimoya I've decided to implement it this way, because then it puts the directories to the right place for sulu-standard and sulu-minimal. I know that setting the context is not very clean, but I don't know any better solution.